pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminTest.java

```diff
@@ -455,9 +455,11 @@ public void properties() throws Exception {
             assertEquals(e.getResponse().getStatus(), Status.NOT_FOUND.getStatusCode());
         }
 
+        AsyncResponse response = mock(AsyncResponse.class);
+
         // Check deleting non-existing property
         try {
-            properties.deleteTenant("non-existing");
+            properties.deleteTenant(response, "non-existing", false);
```

Comment:
Is the test related to this change? I think the test should cover there are namespaces and topics under a tenant, then we delete the tenant forcefully should success.

Reply:
This change adds a force option so the test need pass this option. I can add a test to cover the above situations later.